browser/src/app/LOUtil.ts

```diff
@@ -466,6 +466,12 @@ class LOUtil {
 			insertpage3fwhatlayout3along3d28: 'layout13',
 			insertpage3fwhatlayout3along3d29: 'layout02',
 			insertpage3fwhatlayout3along3d30: 'layout03',
+			graphicfilterinvert: 'graphicfilterinvert',
+			graphicfilterpopart: 'graphicfilterpopart',
```

Comment:
`iconURLAliases` is used to use file with different name than default - no sense to add mapping to the same value